llvm/test/CodeGen/SPIRV/AtomicCompareExchange.ll

```diff
@@ -1,7 +1,7 @@
 ; RUN: llc -O0 -mtriple=spirv32-unknown-unknown %s -o - | FileCheck %s --check-prefix=CHECK-SPIRV
 
 ; CHECK-SPIRV:      %[[#Int:]] = OpTypeInt 32 0
-; CHECK-SPIRV-DAG:  %[[#MemScope_Device:]] = OpConstant %[[#Int]] 1
+; CHECK-SPIRV-DAG:  %[[#MemScope_AllSvmDevices:]] = OpConstant %[[#Int]] 0
```

Comment:
I think it's better to be consistent in terminology of tests vs. source code, and to use here and in other tests CrossDevice instead of AllSvmDevices.

Reply:
Done.